screens/StaticPage/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { CustomButton } from "@/components/ui/Button"
+import CustomText from "@/components/ui/CustomText"
+import CrossMark from "@/components/vectors/CrossMark"
+import { theme } from "@/theme"
+import { StatusBar } from "expo-status-bar"
+import { useState } from "react"
+import {
+  NativeScrollEvent,
+  NativeSyntheticEvent,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from "react-native"
+import { ScrollView } from "react-native-gesture-handler"
+import { NativeStackScreenProps } from "@react-navigation/native-stack"
+import { RootStackParamList } from "@/navigation"
+import { STATIC_PAGE_CONTENTS } from "@/constants/static-page-content"
+
+type StaticPageScreenProps = NativeStackScreenProps<
+  RootStackParamList,
+  "StaticPage"
+>
+
+function StaticPage({ route, navigation }: StaticPageScreenProps) {
+  const [hasScrolledToEnd, setHasScrolledToEnd] = useState(false)
+
+  const handleScroll = (event: NativeSyntheticEvent<NativeScrollEvent>) => {
+    const { layoutMeasurement, contentOffset, contentSize } = event.nativeEvent
+    const isEndReached =
+      layoutMeasurement.height + contentOffset.y >= contentSize.height - 20
+    setHasScrolledToEnd(isEndReached)
+  }
+  const acceptTerms = async () => {
+    navigation.navigate("EntryScreen")
+  }
+  const { page } = route.params
+  return (
+    <View style={styles.container}>
+      <StatusBar style="dark" />
+      <View style={styles.navBar}>
+        <CustomText style={styles.headerText} fontStyle="italic">
+          {STATIC_PAGE_CONTENTS[page].pageTitle}
+        </CustomText>
+        <TouchableOpacity
+          onPress={async () => {
+            navigation.navigate("EntryScreen")
+          }}
```

Comment:
you created a function which does the same thing above, why not call it instead of defining another function here